install/kubernetes/Makefile

```diff
@@ -7,10 +7,11 @@ include ../../Makefile.defs
 VERSION := 1.8.90
```

Comment:
not related to this PR, I just saw this line, this can be addressed separately.

Reply:
https://github.com/cilium/cilium/pull/13863